src/bundle/Resources/public/scss/_card.scss

```diff
@@ -60,6 +69,73 @@
     }
 }
 
+.ez-admin-edit {
+    .ez-card {
+        .form-group,
+        .ez-card--fieldtype-container {
+            .ez-label {
```

Comment:
What do you think about making it a generic class. I mean not nested inside another classes.
Just something like this:
```scss
.ez-label {
    margin-bottom: 0;
    font-weight: 700;
    &:after {
        content: ': ';
    }
}
```
They same rule I would apply for the `.required` class.

Reply:
It would be great (I considered that option too). It will simplify more the process.